pkg/controller/restics.go

```diff
@@ -165,6 +165,27 @@ func (c *StashController) runResticInjector(key string) error {
 		d := obj.(*api.Restic)
 		fmt.Printf("Sync/Add/Update for Restic %s\n", d.GetName())
 
+		if d.Spec.Type == api.BackupOffline {
```

Comment:
I think this should be done at the start of operator and we should check that the image exists and exit if the image is missing. To check image existence use,
https://github.com/k8sdb/apimachinery/blob/3f92c150ed3bedd0d2d19bcc3856360937d06033/pkg/docker/checks.go#L13